apps/auth/main.py

```
@@ -0,0 +1,40 @@
+
+
+@app.get("/frontendinfo", summary="Get frontend info")
+def frontendinfo():
+    return {
+        "PASSWORD_MIN_LENGTH": PASSWORD_MIN_LENGTH,
+        "PASSWORD_MAX_LENGTH": PASSWORD_MAX_LENGTH,
+    }
+
+
+@app.post("/login")
+def login(data: models.LoginData):
```

Ruff (0.8.2)

Missing return type annotation for public function `login`

(ANN201)